File: distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/strategy/AssignmentStrategies.java
package ru.fix.distributed.job.manager.strategy;

public class AssignmentStrategies {
    public static final AssignmentStrategy EVENLY_SPREAD = new EvenlySpreadAssignmentStrategy();
    public static final AssignmentStrategy RENDEZVOUS = new RendezvousHashAssignmentStrategy();
    public static final AssignmentStrategy DEFAULT = new EvenlySpreadAssignmentStrategy();
}

File: distributed-job-manager/src/main/kotlin/ru/fix/distributed/job/manager/strategy/AssignmentStrategies.kt
package ru.fix.distributed.job.manager.strategy

class AssignmentStrategies {
    companion object {
        @JvmField
        val EVENLY_SPREAD: AssignmentStrategy = EvenlySpreadAssignmentStrategy()
        @JvmField
        val RENDEZVOUS: AssignmentStrategy = RendezvousHashAssignmentStrategy()
        @JvmField
        val DEFAULT: AssignmentStrategy = EvenlyRendezvousAssignmentStrategy()
        @JvmField
        val EVENLY_RENDEZVOUS: AssignmentStrategy = EvenlyRendezvousAssignmentStrategy()
    }
}